utils/payload_utils.py

```diff
@@ -0,0 +1,32 @@
+import re
+
+
+class PayloadUtils:
+
+    def extract_testing_status(description):
+        if isinstance(description, str):  # Check if description is a string
+            match = re.search(r"TESTING_STATUS: \[([^\]]+)\]", description)
+            if match:
+                return match.group(1)
+        return None  # Return No
+
+    def extract_testing_recommendation(description):
+        if isinstance(description, str):  # Check if description is a string
+            match = re.search(r"QA_RECOMMENDATION: \[([^\]]+)\]", description)
+            if match:
+                return match.group(1)
+        return None  # Return No
+
+    def extract_build_name(name):
+        if isinstance(name, str):  # Check if description is a string
+            match = re.search(r"Build Validation sign-off - (.+)", name)
```

Comment:
We may wish to add some string sanitizing (already available in string utils) before doing a comparison with what's there.  In general SV copies and pastes the template into the Milestone textarea, however, if they were to use 1 capital letter in place of a lower case, the `re.search` would miss.